quickwit-indexing/src/actors/packager.rs

```diff
@@ -48,20 +49,21 @@ use crate::models::{
 pub struct Packager {
     actor_name: &'static str,
     uploader_mailbox: Mailbox<PackagedSplitBatch>,
-    /// The special field for extracting tags.
-    tags_field: Field,
+    /// A list of tag fields specified in the index config and their
```

Comment:
/// List of tag fields `(name: str, field: Field)` defined in the index config.

Reply:
/// List of tag fields ([Vec<NamedField>]) defined in the index config.